zcash/src/lib.rs

```diff
@@ -12,18 +12,22 @@ extern crate serde;
 extern crate serde_json;
 extern crate sha2;
 
-pub mod address;
+extern crate model;
 
-pub mod builder;
+pub mod address;
+pub use self::address::*;
 
 pub mod network;
 pub use self::network::*;
 
-pub mod privatekey;
-pub use self::privatekey::*;
+pub mod private_key;
+pub use self::private_key::*;
 
-pub mod utils;
-pub use self::utils::*;
+pub mod public_key;
+pub use self::public_key::*;
 
-pub mod wallet;
-pub use self::wallet::*;
\ No newline at end of file
+//pub mod utils;
```

Comment:
if `pub mod utils` is commented out you probably aren't using `utils` so you can just delete it from this lib